bakery/src/tasks/checksum-book.js

```diff
@@ -0,0 +1,39 @@
+const dedent = require('dedent')
+
+const task = () => {
+  return {
+    task: 'checksum book',
+    config: {
+      platform: 'linux',
+      image_resource: {
+        type: 'docker-image',
+        source: {
+          repository: 'openstax/cops-bakery-scripts',
+          tag: 'dev-marvin'
```

Comment:
Did you mean to leave this tag here?

Reply:
no this tag will be removed on final merge.